dojo/tools/securecodebox/parser.py

```diff
@@ -0,0 +1,77 @@
+import hashlib
+import json
+from dojo.models import Endpoint, Finding
+import hyperlink
+
+
+class SecureCodeBoxParser(object):
+    """
+    importing Findings generated by https://github.com/secureCodeBox/secureCodeBox
+    """
+
+    def get_scan_types(self):
+        return ["SecureCodeBox Findings Import"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return "SecureCodeBox Findings Import"
+
+    def get_description_for_scan_types(self, scan_type):
+        return "SecureCodeBox Findings file in JSON format can be imported."
+
+    def get_findings(self, file, test):
+        tree = json.load(file)
+        dupes = dict()
+        for node in tree:
+            finding = Finding(
+                static_finding=True, dynamic_finding=False, active=False, verified=False)
+            self.set_finding_base_info(finding, node, test)
+            finding_unique_string = str(node['location'] +
+                                        finding.description + finding.title)
+            dupe_key = hashlib.md5(
+                finding_unique_string.encode('utf-8')).hexdigest()
+            if dupe_key not in dupes:
+                self.set_finding_endpoint(finding, node)
+                dupes[dupe_key] = finding
```

Comment:
I thing you don't need this part. The SCB format have an ID. we should use this ID directly as the key.